ScoutSuite/frontend/src/utils/Partials/index.js

```diff
@@ -115,6 +117,24 @@ export const renderResourcesAsList = (resources, accessor) => (
   </ul>
 );
 
+/**
+ * Render tags in an unordered list
+ * @param tags 
+ * @returns {HTMLUListElement}
+ */
+export const renderTags = tags => (
```

Comment:
Maybe rename to renderTagsAWS because tags don't work the same way in GCP and Azure (they are just a list of strings).